specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ManagedEnvironmentsDaprResiliencyCircuitBreakerPolicies.json

```diff
@@ -0,0 +1,238 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprResiliencyCircuitBreakerPolicy"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {}
+      },
+      "put": {
+        "tags": [
+          "DaprResiliencyCircuitBreakerPolicies"
+        ],
+        "summary": "Creates or updates a Dapr Component.",
+        "description": "Creates or updates a Dapr Component in a Managed Environment.",
+        "operationId": "DaprResiliencyCircuitBreakerPolicies_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "policyName",
+            "in": "path",
+            "description": "Name of the Dapr resiliency circuit breaker policy.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "daprResiliencyCircuitBreakerPoliciesEnvelope",
+            "in": "body",
+            "description": "Configuration details of the Dapr Policy.",
+            "required": true,
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprResiliencyCircuitBreakerPolicy"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprResiliencyCircuitBreakerPolicy"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "x-ms-examples": {}
+      },
+      "delete": {
+        "tags": [
+          "DaprResiliencyCircuitBreakerPolicies"
+        ],
+        "summary": "Delete a Dapr Component.",
+        "description": "Delete a Dapr Component from a Managed Environment.",
```

Comment:
>Dapr [](http://example.com/codeflow?start=33&length=4)

Expand the term Dapr atleast once at the top of the file for context

Reply:
This is the context, https://docs.dapr.io/operations/resiliency/policies/